### PR TITLE
VR-10117, 10118: Set and get lock level on model versions

### PR DESCRIPTION
**Pull request by convoliution**
Feature-complete; test-writing in progress

### CODE REVIEW

File: client/verta/docs/api/api/lock.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.. lock-api:
+
+Lock
+====
+
+.. note::
+    .. versionadded:: 0.17.3
+
+.. autoclass:: verta.lock.Closed
```
Comment (conradoverta) [inline client/verta/docs/api/api/lock.rst]: `lock` as a top level package is very weird to me. I was expecting `verta.registry.lock` or something like that. Sorry if I missed this in the original proposal

Comment (convoliution) [reply client/verta/docs/api/api/lock.rst]: Oops that's a typo! It is supposed to be `verta.registry.lock`
Thanks!

**Review by conradoverta — APPROVED** [review]
LGTM

**Comment by convoliution** [comment]
I shall write the tests separately!

**Pull request merged**